src/test/app/Offer_test.cpp

```diff
@@ -4414,7 +4415,8 @@ class Offer_test : public beast::unit_test::suite
             env(txn, ter(temDISABLED));
         }
 
-        auto const fsPlus = with_features(fs) | with_features(featureTickSize);
+        auto const fsPlus =
+            featuresWoTickSize | with_only_features(featureTickSize);
```

Comment:
I'd remove this `fsPlus` and change the caller to `testTickSize(features | with_only_features(featureTickSize)`.